lib/main/utils/twake_app_manager.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/model/composer_overlay_state.dart';
+
+class TwakeAppManager {
+  ComposerOverlayState? _composerOverlayState;
```

Comment:
why we need this to manage composer state? why not only base on the cache storage to verify it?

Reply:
> why we need this to manage composer state? why not only base on the cache storage to verify it?

When we open a new composer, we haven't saved it yet so we can't use cache storage to verify. We can only rely on a variable in memory to trigger it.